Formula/gpredict.rb

```diff
@@ -0,0 +1,29 @@
+class Gpredict < Formula
+  desc "Real-time satellite tracking/prediction application"
+  homepage "http://gpredict.oz9aec.net/"
+  url "https://downloads.sourceforge.net/project/gpredict/Gpredict/2.0/gpredict-2.0.tar.gz"
```

Comment:
Please update to latest (version 2.2.1), and use the GitHub download URL (https://github.com/csete/gpredict/releases/download/v2.2.1/gpredict-2.2.1.tar.bz2)